tests/appsec/iast/sink/test_xss.py

```diff
@@ -0,0 +1,38 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+import pytest
+from utils import context, coverage, released
+from ..iast_fixtures import SinkFixture
+
+if context.library == "cpp":
```

Comment:
Can you use the new manifest file for this : `manifests/cpp.yml` ? 

(Same for the `@released` below)